src/Plugin.ts

```diff
@@ -16,6 +16,7 @@ export interface SaveOptions {
 
 export interface RecordOptions {
   content: boolean;
+  excludePaths: string[];
```

Comment:
I attempted to pass a filtering function or RegExp type, which I felt would be more flexible than strings; however, it seems that Cypress has some trouble passing through datatypes like this to the node process that runs Cypress tasks. These attributes always had a value of undefined.

Any experience with this?

Reply:
I guess there's no solution. Cypress will explicitly serializable all arguments which you pass to the task, types such as functions and regex will be omitted (https://docs.cypress.io/api/commands/task#Argument-should-be-serializable).